uvicorn/config.py

```diff
@@ -112,6 +112,20 @@ def create_ssl_context(certfile, keyfile, ssl_version, cert_reqs, ca_certs, ciph
     return ctx
 
 
+def ipvXmessage(version=None):
+    if version == 6:
+        ip_repr = "%s://[%s]:%d"
+    elif version == 4:
```

Comment:
Having missed `version` argument, we'll get a `NameError` below, as `ip_repr` won't be defined.

I'd replace this `elif` by else:
```suggestion
    else:
```

(moreover I'd introduce an enum there, but it's might be an over-engineering)

Reply:
Let me know what you think @cdeler 
I didn't find enums for the 4 or 6 version in the std lib ipaddress but I may have searched badly so I created one.
The only thing missing here is that the ipv6 link is not clickable but I guess it's a shell issue, will check that separetly

Reply:
It's fine, it's what I meant about the enum.